pages/common/kdeconnect-cli.md

```diff
@@ -0,0 +1,36 @@
+# kdeconnect-cli
+
+> KDE Connect CLI.
+> More information: <https://kdeconnect.kde.org>.
+
+- List all devices:
+
+`kdeconnect-cli --list-devices`
+
+- List all reachable devices:
+
+`kdeconnect-cli --list-available`
+
+- Request pairing with a specific device, specifying its ID:
+
+`kdeconnect-cli --pair --device {{device_id}}`
+
+- Request pairing with a specific device, specifying its name:
+
+`kdeconnect-cli --pair --name {{device_name}}`
+
+- Ring a specific device:
+
+`kdeconnect-cli --ring --name {{device_name}}`
+
+- Send an SMS with an optional attachment to a specific number:
```

Comment:
If it has an attachment is an MMS, right? Maybe we should let it clear for the cost 🪙

Reply:
MMS cost isn't a problem everywhere. Also we already mention that it is optional.